<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#lineAddres[state = &quot;0&quot;]{
	color: rgb(255, 74, 51);
}
#lineAddres[state = &quot;1&quot;]{
	color:rgb(102, 255, 51);
}
#WeveDisplay{
	background-color: rgb(239, 255, 172);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>182</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Порт</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinPort">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="ConnectToServer">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="TypeSignal">
       <item>
        <property name="text">
         <string>cos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>atan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>acos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>asin</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineAddres">
       <property name="inputMask">
        <string>000.000.000.000;_</string>
       </property>
       <property name="state" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Ip">
       <property name="text">
        <string>Ip</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="SendRequest">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Spam">
       <property name="text">
        <string> ------</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>127.0.0.1:1300</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="WeveDisplay">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFileLinck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/Users/malysevdanil/Documents/Projects/P2/Client/sifgnal_data2.txt</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>456</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
